CHANGES.md

```diff
@@ -5,13 +5,15 @@ Change Log
 
 ##### Additions :tada:
 * Added optional `index` parameter to `PrimitiveCollection.add`. [#8041](https://github.com/AnalyticalGraphicsInc/cesium/pull/8041)
+* Updated to render at the browser max resolution. Set `viewer.resolutionScale = 1.0 / window.devicePixelRatio` to render at lower pixel density. [#8082](https://github.com/AnalyticalGraphicsInc/cesium/issues/8082)  
```

Comment:
I'm not sure "browser max resolution" would get the point across, maybe something like: "Cesium now renders at native device resolution by default instead of CSS pixel resolution, to go back to the old behavior...xxxx"